acquisitions/src/main/resources/thunderjet/mod-invoice/features/check-invoice-line-validation-with-adjustments.feature

```diff
@@ -0,0 +1,71 @@
+Feature: Check invoiceLine validation with  adjustments
```

Comment:
Please add jira story name, and I think this tests should be renamed to something like this: check-invoice-validation-for-adjustments-not-fails-with-correct-adjustments

Reply:
or if we remain existing name - we need to add 2 scenarios, the first with failed validation, the second - with passed

Reply:
We removed the logic of validation from invoiceLine , so i dont think faliure will occur in any case. So renaming it to suggested name.

Reply:
okay, you added failure scenario, thanks